### PR TITLE
Tony's feature

### PR DESCRIPTION
**Pull request by chitalian**

### CODE REVIEW

File: examples/DDL.txt [hunk 1]
```diff
@@ -0,0 +1,26 @@
+create table sisu_facts (
```
Comment (chitalian) [inline examples/DDL.txt:1]: Should this file be a `.sql`?

**Comment by chitalian** [comment]
Hey @realtonyyoung can you run flake on these changes?

**Comment by chitalian** [comment]
Can you also add a README in the examples directory with some example runs. 

I think we should also get rid of `examples/Sisu Facts.pptx` from the repo and not commit compiled binaries

File: examples/DDL.txt (hunk 1 above)
Comment (realtonyyoung) [reply examples/DDL.txt:1]: Fixed

**Comment by realtonyyoung** [comment]
> Hey @realtonyyoung can you run flake on these changes?

Done and fixed issues except long line issues

**Comment by realtonyyoung** [comment]
> Can you also add a README in the examples directory with some example runs.
> 
> I think we should also get rid of `examples/Sisu Facts.pptx` from the repo and not commit compiled binaries

Removed the PPTX

**Pull request merged**